scripts/ci-is-generated-in-sync.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+readonly REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+
+cd "${REPO_ROOT}"
+
+make generate
+diff=$(git diff --name-only)
+
+echo "Following files need to be regenerated. Please run 'make generate' and check them in"
+echo ${diff}
```

Comment:
This needs to exit non-zero if a diff exists for the related test job to fail.

Reply:
I woke up realizing I missed this! 🤣 
I'll fix it up